EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+[!!Exports](../UserInterface/01_Exports.md)
+
+
+# Manage header file settings
+
+The following procedures show by using an example how to specify the fields required by your business partner. As example the D07A INVOIC message is used.  
+
+>[INFO] The following procedures are examples only of how to specify specific header segments of the EDIFACT message. Follow the guidelines of your business partner when specifying your EDIFACT message.  
+
+You can add constant values as well as strings that are to be determined via attribute mapping. Additionally, you can specify conditions for the output of specific segments.
+
+## Get header file segments
+
+Get the header file segments of an EDIFACT message in order to be able to fill them with data. By doing this, you add the whole message structure of the specified message type.
+
+> [INFO] The UNA, UNB and UNH segments are generated automatically. Therefore, the definition starts with the S_BGM segment.
+
+#### Prerequisites
+
+- You have the message specification of your business partner at hand.
+- You have created the basic definition settings for this type of message and business partner, see [Create basic definition settings](./01_ManageDefinitions.md#create-basic-definition-settings).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab*
+
+![Exporter definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Exporter definitions]")
+
+1. Click the definition for which you want to specify the header file settings.   
+    The *Edit definition "definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+    ![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEdit.png "[Edit definition]")
+
+2. Click the *File settings* tab.   
+    The *File settings* tab is displayed.
+
+     ![File settings tab](../../Assets/Screenshots/EDI/Operation/ExportDefDefineFileSettings.png "[File settings tab]")
+
+3. If you have not already done so, click the *Release* dropdown list and select the required release.
+
+4. If you have not already done so, click the *Message type* dropdown list and enter the required message type.
+
+5. Click the ![Get](../../Assets/Icons/Download.png "[Get]") [GET] button.   
+    The message structure is displayed below. The message name is highlighted in red because it is still empty.
+
+    ![Message](../../Assets/Screenshots/EDI/Operation/ExportDefMessNew.png "[Message]")
+
+6. Expand the message structure by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button.   
+    The message structure with its single segments is displayed. All segments that contain mandatory content in the EDIFACT standard, are indicated with an M[Number] in italic and bold, for example ***M35***.    
+    The ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button has changed to an ![Expanded](../../Assets/Icons/NodeExpanded.png "[Expanded]") (Expanded) button.
```

Comment:
Ich denke, diese Info gehört eher in UI Kapitel hin.

Reply:
ne